cli/tests/unit/data/node.txt

```diff
@@ -1,2 +1,2 @@
-HOSTNAME       IP                          ID                    
-dcos-01   172.17.8.101  20150630-004309-1695027628-5050-1649-S0  
\ No newline at end of file
+HOSTNAME       IP                          ID                    TYPE   
+dcos-01   172.17.8.101  20150630-004309-1695027628-5050-1649-S0  slave  
```

Comment:
Also the hostnames still seem to be `master.mesos` so I think it makes sense for the type to be `master` or `master (leader)` like this:

```
$ dcos node
   HOSTNAME         IP                          ID                     TYPE
 10.71.90.121  10.71.90.121  80527015-4faa-4c42-ab6b-7e36c4a22ee5-S0   agent
 10.71.91.112  10.71.91.112  80527015-4faa-4c42-ab6b-7e36c4a22ee5-S5   agent
 10.71.91.137  10.71.91.137  80527015-4faa-4c42-ab6b-7e36c4a22ee5-S12  agent
 10.71.91.25   10.71.91.25   80527015-4faa-4c42-ab6b-7e36c4a22ee5-S4   agent
 10.71.91.37   10.71.91.37   80527015-4faa-4c42-ab6b-7e36c4a22ee5-S1   agent
 10.71.91.49   10.71.91.49   80527015-4faa-4c42-ab6b-7e36c4a22ee5-S11  agent
 10.71.91.71   10.71.91.71   80527015-4faa-4c42-ab6b-7e36c4a22ee5-S2   agent
 10.71.91.73   10.71.91.73   80527015-4faa-4c42-ab6b-7e36c4a22ee5-S7   agent
  10.71.91.8    10.71.91.8   80527015-4faa-4c42-ab6b-7e36c4a22ee5-S14  agent
 10.71.91.84   10.71.91.84   80527015-4faa-4c42-ab6b-7e36c4a22ee5-S3   agent
master.mesos.  10.71.91.155                    N/A                     master
master.mesos.  10.71.91.59                     N/A                     master
master.mesos.  10.71.91.68     f6365617-8938-4968-8421-e80d1ec33143    master (leader)
```

but let me know if you have other ideas.